### DISCUSSION TITLE
I got error when use useFieldArray append with material ui select

### DISCUSSION THREAD

**Discussion by zealgt** [react-hook-form/react-hook-form#6705] [Q&A]
react hook form version: 6.15.8
material-ui version: 4.12.3

Please help me, I got error from Material UI Select component (Cannot read properties of null (reading 'focus')) when I try to call append function from useFieldArray hook.

Codesandbox link
https://codesandbox.io/s/react-hook-form-usefieldarray-forked-h248x?file=/src/index.js

**Comment by bluebill1049** [comment]
how could I reproduce this issue? I think it's mostly due to the input `ref` is not been assigned correctly.

**Reply by zealgt** [reply]
Sorry I forgot to save code, so you can reproduce by click "Append" button you will see the error page.

By the way, I implemented this code with other component and it work fine except use with array field
